app/views/case_studies/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :title, "#{@case_study_presenter.page_title} - #{I18n.t("formats.#{@content_item.document_type}", count: 1)} - GOV.UK" %>
+
+<% content_for :extra_headers do %>
+  <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+<% end %>
+
+<main role="main" id="content" class="case-study" <%= lang_attribute(@content_item.locale) %>>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds responsive-top-margin">
+      <%= render 'govuk_publishing_components/components/title', {
+        title: content_item.title,
+        context: I18n.t("formats.#{content_item.document_type}", count: 1),
+        context_locale: t_locale_fallback("formats.#{content_item.document_type}", count: 1),
+        average_title_length: "long",
+      } %>
+    </div>
+    <%= render 'shared/translations' %>
+    <div class="govuk-grid-column-two-thirds">
+      <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+    </div>
+  </div>
+
+  <hr class="govuk-section-break govuk-section-break--s govuk-section-break--visible" aria-hidden="true">
```

Comment:
The border is defined in https://github.com/alphagov/government-frontend/blob/main/app/assets/stylesheets/helpers/_publisher-metadata-with-logo.scss so I think once it's added in https://github.com/alphagov/frontend/pull/4372/commits/26a483dd6224a6d6edbda1925b80c8eeb07d16c1 this can be removed.